app/src/androidTest/java/org/systers/mentorship/SignUpActivityTest.kt

```diff
@@ -254,6 +254,6 @@ class SignUpActivityTest {
         onView(withId(R.id.btnSignUp)).perform(scrollTo(), click())
 
         onView(withId(com.google.android.material.R.id.snackbar_text))
-                .check(matches(withText(EMAIL_ALERADY_REGISTERED)))
+                .check(matches(withText(EMAIL_ALREADY_REGISTERED)))
     }
 }
```

Comment:
Add a new line here.

Reply:
Is it ok now?

Reply:
Each file should have a blank line at the end

Reply:
Do you want to say that here at line 260 should be and that should be blank?

Reply:
Yes